packages/itwin/property-grid/scripts/start-test-viewer-no-watch.js

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+const crossEnv = require("cross-env");
```

Comment:
maybe we could enhance existing `start-test-viewer.js` to accept a `--watch` argument?

Reply:
Changed